packages/kit/src/runtime/server/page/utils.js

```diff
@@ -0,0 +1,11 @@
+/** @param {URL} url */
+export function create_url_proxy(url) {
```

Comment:
Perhaps a different name for this? The current one doesn't suggest anything about this just being used for prerendering.

Reply:
done